File: Assets/My Assets/Scripts/UI/Narrative/Cutscene Handler.js
﻿var ourSteps : GameObject[];
var interval : int = 0;
var nextScene : String;
var loadingText : GameObject;
var myTimer : float = 0;
function Update(){
	myTimer+=Time.deltaTime;
	if(myTimer >= 2){
		if(Input.anyKeyDown){
			iterateArray();
		}
	}
}
function Start(){
	if(PlayerPrefs.GetInt("EffectsCheck") == 0){
		GameObject.FindGameObjectWithTag("MainCamera").GetComponent("Tube").enabled = false;
	}
}
function iterateArray(){
	if(interval+1 < ourSteps.Length){
		interval+=1;
	}
	else{
		var myText = Instantiate(loadingText,this.transform.position,Quaternion.identity);
		myText.transform.SetParent(GameObject.Find("Canvas").transform,false);
		Application.LoadLevel(nextScene);
	}
	ourSteps[interval-1].active = false;
	ourSteps[interval].active = true;
}

File: Assets/My Assets/Scripts/UI/Narrative/CutsceneHandler.cs
using UnityEngine;
using System.Collections;

[System.Serializable]
public partial class CutsceneHandler : MonoBehaviour
{
    public GameObject[] ourSteps;
    public int interval;
    public string nextScene;
    public GameObject loadingText;
    public float myTimer;
    public virtual void Update()
    {
        this.myTimer = this.myTimer + Time.deltaTime;
        if (this.myTimer >= 2)
        {
            if (Input.anyKeyDown)
            {
                this.iterateArray();
            }
        }
    }

    public virtual void Start()
    {
        if (PlayerPrefs.GetInt("EffectsCheck") == 0)
        {
            GameObject.FindGameObjectWithTag("MainCamera").GetComponent<Kino.Tube>().enabled = false;
        }
    }

    public virtual void iterateArray()
    {
        if ((this.interval + 1) < this.ourSteps.Length)
        {
            this.interval = this.interval + 1;
        }
        else
        {
            GameObject myText = UnityEngine.Object.Instantiate(this.loadingText, this.transform.position, Quaternion.identity);
            myText.transform.SetParent(GameObject.Find("Canvas").transform, false);
            Application.LoadLevel(this.nextScene);
        }
        this.ourSteps[this.interval - 1].active = false;
        this.ourSteps[this.interval].active = true;
    }

}